packages/@lwc/synthetic-shadow/src/faux-shadow/traverse.ts

```diff
@@ -83,7 +84,7 @@ export function getAllMatches(owner: Element, nodeList: NodeList | Node[]): Arra
     return filteredAndPatched;
 }
 
-export function getFirstMatch(owner: Element, nodeList: NodeList): Element | null {
+export function getFirstMatch(owner: Element, nodeList: NodeList | Element[]): Element | null {
```

Comment:
this type extension seems wrong.

Reply:
This method is called in two places. One with a NodeList and the other with an Element[]. That is the reason for this change.
BTW, the same pattern is in L73 
`export function getAllMatches(owner: Element, nodeList: NodeList | Node[]):`